cmd/verify.go

```diff
@@ -161,21 +169,25 @@ func NewVerifyCmd(config *viper.Viper) *cobra.Command {
 		},
 	}
 
-	cmd.Flags().StringSliceVarP(&opts.ValueFiles, "chart-values", "f", nil, "specify values in a YAML file or a URL (can specify multiple)")
+	settings.AddFlags(cmd.Flags())
+
+	cmd.Flags().StringSliceVarP(&opts.ValueFiles, "chart-values", "F", nil, "specify values in a YAML file or a URL (can specify multiple)")
```

Comment:
any reason for changing short arg from `f` to `F`?

Reply:
To make room for now; I understand the defaults are not good so any ideas on how to name the short options are welcome.